network/src/protocols/wire/handshake/v1/mod.rs

```diff
@@ -113,15 +113,20 @@ impl fmt::Display for ProtocolId {
 }
 
 //
-// SupportedProtocols
+// ProtocolIdSet
 //
 
-/// A bit vector of supported [`ProtocolId`]s.
+/// A compact representation for a set of [`ProtocolId`]s. Internally, this is a
+/// bitvec which supports at most 256 bits.
+///
+/// These sets are sent over-the-wire in the initial [`HandshakeMsg`] to other
+/// DiemNet peers in order to negotiate the set of common supported protocols for
+/// use on a new DiemNet connection.
 #[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
 #[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
-pub struct SupportedProtocols(bitvec::BitVec);
+pub struct ProtocolIdSet(bitvec::BitVec);
 
-impl SupportedProtocols {
+impl ProtocolIdSet {
```

Comment:
I really appreciate having a name that describes what it is rather than `SupportedProtocols`